src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -947,7 +946,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
 
         const auto actualNewSize = _actualFont.GetSize();
-        _FontSizeChangedHandlers(actualNewSize.width, actualNewSize.height, initialUpdate);
+        _FontSizeChangedHandlers(*this, *winrt::make_self<FontSizeChangedArgs>(actualNewSize.width, actualNewSize.height));

```

Comment:
`*make_self<T>()` is equivalent to `make<T>` when T's default interface is the same as the one you're converting to.

For new code, we should prefer the clearer one :)

Reply:
I forgot that make works even on a non projected ctor